metrics/src/main/java/io/opentelemetry/example/metrics/ExponentialHistogramExample.java

```diff
@@ -0,0 +1,82 @@
+package io.opentelemetry.example.metrics;
+
+import static io.opentelemetry.api.common.AttributeKey.stringKey;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.LongHistogram;
+import io.opentelemetry.api.metrics.Meter;
+import io.opentelemetry.exporter.otlp.metrics.OtlpGrpcMetricExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.metrics.Aggregation;
+import io.opentelemetry.sdk.metrics.InstrumentSelector;
+import io.opentelemetry.sdk.metrics.SdkMeterProvider;
+import io.opentelemetry.sdk.metrics.View;
+import io.opentelemetry.sdk.metrics.export.PeriodicMetricReader;
+import java.time.Duration;
+import java.util.Random;
+
+public class ExponentialHistogramExample {
```

Comment:
I think that you should take part of your PR description and just add in a little javadoc for this example class...just to give the casual/quick reader some intro knowledge about what they're looking at.